src/common/components/repositories-home/index.js

```diff
@@ -2,18 +2,35 @@ import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 
 import { fetchUserSnaps } from '../../actions/snaps';
+import { fetchBuilds } from '../../actions/snap-builds';
 import { Anchor } from '../vanilla/button';
 import RepositoriesList from '../repositories-list';
 import styles from './repositories-home.css';
 
 class RepositoriesHome extends Component {
+  fetchData(props) {
```

Comment:
Currently it's `RepositoriesHome` that fetches the data for `/dashboard`.
Maybe in future it would be worth moving it down to particular component that need that data (link RepositoriesList).